instance/management/commands/instance_statistics_csv.py

```diff
@@ -0,0 +1,221 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015-2019 OpenCraft <contact@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Instance app - Activity report management command
+"""
+
+# Imports #####################################################################
+
+from configparser import ConfigParser
+import csv
+from datetime import datetime
+import os
+import sys
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+
+from instance import ansible
+from instance.models.openedx_instance import OpenEdXInstance
+from registration.models import BetaTestApplication
+
+
+# Classes #####################################################################
+
+
+class Command(BaseCommand):
+    """
+    Logs_activity_csv management command class
+    """
+    help = (
+        'Generates a CSV containing basic activity information about all app servers'
+        ' (numbers for hits, distinct hits, users, and courses).'
+    )
+
+    LOGS_SERVER = 'logs.opencraft.com'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--qualified-domain',
+            '-q',
```

Comment:
@pcockwell, wouldn't this be better?

```suggestion
            '--domain',
            '-d',
```

Reply:
There is already a `-d` argument for `--days`, which is why I went with `-q` and `--qualified-domain`. I'm open to alternative argument flags though

Reply:
@pcockwell, the flag for `days` is likely not going to be useful/needed because that doesn't allow specifying the start/end dates, which will be needed for a report for the last completed month.